### PR TITLE
Remove calls to PortableRemoteObject.narrow

### PR DESCRIPTION
**Pull request by dblevins**
The refactor was done by adding a javax.rmi.PortableRemoteObject class
to the project with a `narrow` method that simply returned the original
object.  The narrow method was then inlined.

Intellij will clean unused imports even if they do not relate to the
refactor, so there are a lot of .* imports that are also deleted.
I hope this is not an issue.

I recommend we run at least these sections:

```
./src/com/sun/ts/tests/ejb/ee/bb/entity/bmp/clientviewtest
./src/com/sun/ts/tests/ejb/ee/bb/entity/cmp/clientviewtest
./src/com/sun/ts/tests/ejb/ee/bb/entity/cmp20/clientviewtest
./src/com/sun/ts/tests/ejb/ee/bb/entity/lrapitest
./src/com/sun/ts/tests/ejb/ee/bb/session/lrapitest
./src/com/sun/ts/tests/ejb/ee/bb/session/stateful/clientviewtest
./src/com/sun/ts/tests/ejb/ee/bb/session/stateless/clientviewtest
```

### CODE REVIEW

**Comment by RohitKumarJain** [comment]
You might need to consider updating the Copyright (c) year for all the impacted the files to the current year if it's not updated. 
Ex: https://github.com/eclipse-ee4j/jakartaee-tck/blob/c0336f8f55e7eb2b74f51bd77756c423c6330980/src/com/sun/ts/lib/util/TSNamingContext.java file needs to updated with Copyright (c) 2007, 2020 from Copyright (c) 2007, 2018, as this file has changed recently.

File: src/com/sun/ts/lib/util/TSNamingContext.java [hunk 1]
```diff
@@ -53,7 +51,7 @@ public TSNamingContext(Properties pp) throws Exception {
    */
   public Object lookup(String s, Class c) throws Exception {
     Object o = lookup(s);
-    return c == null ? o : PortableRemoteObject.narrow(o, c);
+    return c == null ? o : o;
```
Comment (scottmarlow) [inline src/com/sun/ts/lib/util/TSNamingContext.java:54]: If `c` is not null, should we throw a ClassCastException failure if the returned object is not an instance of `Class c`?

**Comment by scottmarlow** [comment]
https://ci.eclipse.org/jakartaee-tck/job/jakartaee-tck-scottmarlow/job/interop/2/console is building the TCK and running tests webservices12/ejb ejb ejb30 assembly xa/ee/xresXcomp2 xa/ee/resXcomp1 appclient/deploy/ejblink/casesens  rmiiiop/ee/objecttests, if they pass, I'll do a full run.

**Comment by scottmarlow** [comment]
Added `future` label to reflect that this work will be done after EE 9 when we do the JDK11 work.

**Pull request closed without merging**